### PR TITLE
ADD: New Endpoints and Controller to: ArtesianWell, Hidrometer and WaterTankLevel

### PR DESCRIPTION
**Pull request by LCStuber**
New Endpoints and Controller to:

- ArtesianWell
- Hidrometer
- WaterTankLevel

Also, some correction to Database normalization and connection to the SmartLights.

*README was updated for the above endpoints.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -1,18 +1,22 @@
...
+	api := r.Group("/api/timeseries/v0.2/smartcampusmaua")
 	{
-		api.GET("", controller.GetSmartLights)
-		api.GET("deviceName/:nodename", controller.GetSmartLightbyNodeName)
-		api.GET("deviceId/:devEUI", controller.GetSmartLightbyDevEUI)
+		api.GET("SmartLights", controller.GetSmartLights)
+		api.GET("SmartLights/deviceName/:nodename", controller.GetSmartLightbyNodeName)
+		api.GET("SmartLights/deviceId/:devEUI", controller.GetSmartLightbyDevEUI)
+
+		api.GET("WaterTankLevel", controller.GetWaterTankLevel)
```
Comment (rogeriocassares) [inline main.go]: Modify to "WaterTankLevels" since it returns an array of WaterTankLevel sensors.

File: database/database.go [hunk 2]
```diff
@@ -2,14 +2,14 @@ package database
...
-	"os"
+
 	"github.com/InfluxCommunity/influxdb3-go/influxdb3"
 )
 
 func ConnectToDB() (*influxdb3.Client, error) {
-	url := os.Getenv("INFLUXDB_URL")
-	token := os.Getenv("INFLUXDB_TOKEN")
-	database := os.Getenv("INFLUXDB_DATABASE")
+	url := "https://us-east-1-1.aws.cloud2.influxdata.com"
```
Comment (rogeriocassares) [inline database/database.go]: Please, do not change how env are implemented for security and operate in production in a differebt db of development.

File: controller/watertank_controller.go [hunk 3]
```diff
@@ -0,0 +1,235 @@
+package controller
```
Comment (rogeriocassares) [inline controller/watertank_controller.go]: please let the name of this file as waterTankLevels_controller.go and change the smartlight_controller.go to smartLights_controller.go.

In description of this PR ArtesianWell and Hidrometer were not find to PR.

File: controller/smartlight_controller.go [hunk 4]
```diff
@@ -50,37 +50,37 @@ func GetSmartLights(c *gin.Context) {
...
-					"rxInfo_altitude_0": value["rxInfo_altitude_0"],
-					"rxInfo_latitude_0": value["rxInfo_latitude_0"],
-					"rxInfo_loRaSNR_0": value["rxInfo_loRaSNR_0"],
-					"rxInfo_longitude_0": value["rxInfo_longitude_0"],
-					"rxInfo_rssi_0": value["rxInfo_rssi_0"],
-					"txInfo_dataRate_spreadFactor": value["txInfo_dataRate_spreadFactor"],
-					"txInfo_frequency": value["txInfo_frequency"],
-					"data_humidity": value["data_humidity"],
-					"data_temperature": value["data_temperature"],
+				"data_counter_0d_0":            value["data_counter_0d_0"],
```
Comment (rogeriocassares) [inline controller/smartlight_controller.go]: Please let all "data_VALUES" in sequence, alfabetic order.

**Review by rogeriocassares — CHANGES_REQUESTED** [review]
Please review to PR again.

Thanks!

File: controller/watertank_controller.go (hunk 3 above)
Comment (LCStuber) [reply controller/watertank_controller.go]: File names changed as requested, the ArtesianWell and Hidrometer are already in the PR.

File: main.go (hunk 1 above)
Comment (LCStuber) [reply main.go]: Changed as requested

File: database/database.go (hunk 2 above)
Comment (LCStuber) [reply database/database.go]: During the development we saw that we unfortunately have commited it... We revoked the key present, and returned to the .env specification in a future commit!

File: controller/smartlight_controller.go (hunk 4 above)
Comment (LCStuber) [reply controller/smartlight_controller.go]: Changed as requested

**Review by LCStuber — COMMENTED** [review]
Changes made!

File: database/database.go (hunk 2 above)
Comment (LCStuber) [reply database/database.go]: *This PR is already with the updated commit with the .env structures

**Review by rogeriocassares — APPROVED** [review]
Ok.

**Pull request merged**